po/es.po

```diff
@@ -0,0 +1,606 @@
+msgid "Play / Pause"
+msgstr "Interpretar / Pausa"
+
+#: assets/ui/media-toc.ui:771
+msgid "Video Streams"
+msgstr "Secuencias de vídeo"
+
+#: assets/ui/media-toc.ui:804
+msgid "Audio Streams"
+msgstr "Secuencias de audio"
+
+#: assets/ui/media-toc.ui:837
+msgid "Text Streams"
+msgstr "Secuencias de texto"
+
+#: assets/ui/media-toc.ui:967
+msgid "mkvmerge text format"
+msgstr "Formato de texto de mkvmerge"
+
+#: assets/ui/media-toc.ui:983
+msgid ""
+"mkvmerge compliant text format.\n"
+"Flat table of contents.\n"
+"Time resolution: 1 ms."
+msgstr ""
+"Formato de texto compatible mkvmerge.\n"
+"Tabla de contenido plana. \n"
+"Unidad de tiempo: 1 ms."
+
+#: assets/ui/media-toc.ui:1017
+msgid "Cue Sheet format"
+msgstr "Formato Cue Sheet"
+
+#: assets/ui/media-toc.ui:1033
+msgid ""
+"Cue Sheet compliant text format.\n"
+"Hierachical table of contents.\n"
+"Time resolution: 1 frame (1/75 s)."
+msgstr ""
+"Formato de texto compatible Cue Sheet.\n"
+"Tabla de contenido jerárquica.\n"
+"Unidad de tiempo: 1 trama (1/75 s)."
+
+#: assets/ui/media-toc.ui:1067
+msgid "Matroska Container"
+msgstr "Contenedor Matroska"
+
+#: assets/ui/media-toc.ui:1083
+msgid ""
+"Binary audio, video & text media container.\n"
+"Hierachical table of contents.\n"
+"Time resolution: 1 ns."
+msgstr ""
+"Contenedor binario de audio, vídeo y texto.\n"
+"Tabla de contenido jerárquica.\n"
+"Unidad de tiempo: 1 ns.\n"
+
+#. Button label
+#: assets/ui/media-toc.ui:1127
+msgid "Export"
+msgstr "Exportar"
+
+#. Button tooltip
+#: assets/ui/media-toc.ui:1131
+msgid "Export media to selected format"
+msgstr "Exportar archivo multimedia al formato seleccionado"
+
+#: assets/ui/media-toc.ui:1149
+msgid "Export Formats"
+msgstr "Formatos de exportación"
+
+#: assets/ui/media-toc.ui:1267
+msgid "Flac"
+msgstr "Flac"
+
+#: assets/ui/media-toc.ui:1283
+msgid ""
+"Audio lossless compression.\n"
+"Tags support."
+msgstr ""
+"Compresión de audio sin pérdidas.\n"
+"Gestión de etiquetas."
+
+#: assets/ui/media-toc.ui:1335
+msgid "Wave / PCM"
+msgstr "Wave / PCM"
+
+#: assets/ui/media-toc.ui:1351
+msgid ""
+"No compression.\n"
+"Tags support."
+msgstr ""
+"Sin compresión.\n"
+"Gestión de etiquetas."
+
+#: assets/ui/media-toc.ui:1403
+msgid "Opus"
+msgstr "Opus"
+
+#: assets/ui/media-toc.ui:1419 assets/ui/media-toc.ui:1487
+#: assets/ui/media-toc.ui:1555
+msgid ""
+"Audio lossy compression.\n"
+"Tags support."
+msgstr ""
+"Compresión de audio con pérdidas.\n"
+"Gestión de etiquetas."
+
+#: assets/ui/media-toc.ui:1471
+msgid "Vorbis"
+msgstr "Vorbis"
+
+#: assets/ui/media-toc.ui:1539
+msgid "MP3"
+msgstr "MP3"
+
+#. Button label
+#. Name of the perspective
+#: assets/ui/media-toc.ui:1598 assets/ui/media-toc.ui:1979
+msgid "Split"
+msgstr "Dividir"
+
+#. Button tooltip
+#: assets/ui/media-toc.ui:1602
+msgid "Split audio stream and encode to selected format"
+msgstr "Dividir la secuencia de audio y codificarla al formato seleccionado"
+
+#: assets/ui/media-toc.ui:1620
+msgid "Split Formats"
```

Comment:
It seems you used a different form ("dividir") elsewhere to express "split" than this one ("corte"). I understand this a a name when "dividir" is a verb, but would it possible to use the name for "dividir" here?

Reply:
I can change  to "formatos de división" but it sound weird in spanish... can you give me an example  of how this is displayed? is there a list of the formats  after this label? and if yes , which ones? it can help me to take the decision. Thanks!

Reply:
This is the title for the list of audio formats which the user can select to split the media file into audio files. That's the list that starts [here](https://github.com/XTR81/media-toc/blob/d8963fc6041345a02593881899133489a740f44d/po/es.po#L167). In the English version, select the "Split" perspective. It's the title of the list in this perspective.

Reply:
As an example, this is the same purpose as in the 2d description of the application, which you translated as "o **divide** un archivo multimedia en capítulos".